src/components/TodoCard.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ReactComponent as CheckBoxSvg } from '../assets/svg/CheckBox.svg';
+import { ReactComponent as CheckBoxOutlineSvg } from '../assets/svg/CheckBoxOutline.svg';
+import { ReactComponent as DeleteSvg } from '../assets/svg/Delete.svg';
+import useLocalStorage from '../hooks/useLocalStorage';
+
+let TodoCardLayout = styled.div`
+  width: 100%;
+  height: 2.5rem;
+  display: flex;
+  align-items: center;
+  padding: 0.5rem 1rem;
+  gap: 0.5rem;
+  &:hover {
+    background-color: #f5f5f5;
+    cursor: pointer;
+    button {
+      display: flex;
+    }
+  }
+`;
+
+let TodoCardParagraph = styled.p`
+  flex-grow: 1;
+`;
+let CheckboxBox = styled.div`
+  flex-shrink: 0;
+`;
+
+let TodoDeleteButton = styled.button`
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: none;
+  flex-shrink: 0;
+  cursor: pointer;
+  &:hover {
+    background-color: gray;
+  }
+  display: none;
+`;
+
+export default function TodoCard({ isdone, todo, deleteTodo, toggleTodo }) {
```

Comment:
props로 넘겨받을때 구조분해할당하신거 너무 좋아요! 저의 경우에는 2개 이상의 props인 경우 가독성을 위해 아래와 같이 쓰는데요, 기능은 같으니까 참고만하시면 될 것 같아요 ㅎㅎ
```suggestion
export default function TodoCard(props) {
    const {isdone, todo, deleteTodo, toggleTodo} = props;
```